_posts/2020-10-05-Evidence_locker_overview.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+category: Architecture
+title: Evidence locker overview
+---
+
+At the core of the Auditree framework is a [Git][] repository known as the
+evidence locker.  Each Auditree execution environment requires a dedicated
```

Comment:
quote "evidence locker" ?

Reply:
Done in https://github.com/auditree/auditree.github.io/pull/6/commits/406f0ed0a3b7b1b75e25c945c7f52be953d95e0f